scripts/utils/price-utils.js

```diff
@@ -2,6 +2,9 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
   const axios = require("axios")
   const BN = require("bn.js")
   const exchangeUtils = require("@gnosis.pm/dex-contracts")
+  const { Fraction } = require("@gnosis.pm/dex-contracts/src")
+
+  const max128 = new BN(2).pow(new BN(128)).subn(1)
```

Comment:
I thought this value was already declared in another file somewhere. Couldn't we just import it?